packages/react-super-components/src/components/msc-tile/msc-tile.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import i18next from 'i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsChip } from '@ovhcloud/ods-components/chip/react';
+import { OsdsTile } from '@ovhcloud/ods-components/tile/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react';
+
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { useDynamicTranslation } from '../../translation/useDynamicTranslation';
+
+interface Badge {
+  text: string;
+  color: ODS_THEME_COLOR_INTENT;
+}
+
+export interface ScTileProps {
+  href?: string;
+  isExternalHref?: boolean;
+  imgSrc?: string;
+  imgAlt?: string;
+  texts: {
+    title: string;
+    description: string;
+    category: string;
+  };
+  dataTracking?: string;
+  badges?: Badge[];
+  children?: React.ReactElement;
```

Comment:
There is a `React.PropsWithChildren` for this

Reply:
No `PropsWithChildren` return an object with all props { ... props, children } and children also 
[Link here
](https://blog.logrocket.com/react-children-prop-typescript/)